### PR TITLE
Fix for making student ProphetNet for Seq2Seq Distillation

### PR DESCRIPTION
**Pull request by vishal-burman**
# What does this PR do?

Fixes # (issue)
Enables making student model of ProphetNet

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/master/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/master/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/master/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?
@sgugger @LysandreJik

### CODE REVIEW

**Comment by sgugger** [comment]
We're not actively maintaining those examples, so would need an approval from the original author (@sshleifer ) before merging

**Review by sshleifer — APPROVED** [review]

**Comment by sshleifer** [comment]
LGTM!

**Comment by sgugger** [comment]
Thank you both!

**Pull request merged**
